charts/envoy/values.yaml

```diff
@@ -4,12 +4,14 @@ replicaCount: 3
 envoyConfiguration:
   # envoyConfiguration.adminAccessLogPath -- admin log path
   adminAccessLogPath: /dev/stdout
+  # envoyConfiguration.serviceListeners -- list of service name and port
+  serviceListeners: scalar-service:50051,scalar-privileged:50052
```

Comment:
Just a confirmation.
This is not necessary to make it work, right?
(since `scalar-service:50051,scalar-privileged:50052` is the default string specified in envoy.yaml)

Reply:
> This is not necessary to make it work, right?

Yes, it is not a required setting.
This is only written as an example. If that is the case, would it be better to leave this blank and provide an example in the comments?

Reply:
Thank you! I think it's fine! (I think README is updated by this so it's helpful for users.)